### PR TITLE
Adds Paraview 5.9.1-RC1, 5.9.0

### PR DESCRIPTION
**Pull request by vicentebolea**
Hi there, first release candidate of 5.9.1!

This morning we have just released the latest ParaView version.

I have made that changes in the ParaView Spack package which adds this new version .

I have also set explicitly the preferred version to 5.9.0 so that the default version does not change with this addition.

Note that this is a release candidate.

### CODE REVIEW

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ class Paraview(CMakePackage, CudaPackage):
     maintainers = ['chuckatkins', 'danlipsa', 'vicentebolea']
 
     version('master', branch='master', submodules=True)
-    version('5.9.0', sha256='b03258b7cddb77f0ee142e3e77b377e5b1f503bcabc02bfa578298c99a06980d')
+    version('5.9.0-RC1', sha256='bb3f12bd3d31bcb52455e1393e45f81a18754d3c3e8199d51532b0c067dc1595')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/paraview/package.py]: I think there's a typo:
```suggestion
    version('5.9.1-RC1', sha256='bb3f12bd3d31bcb52455e1393e45f81a18754d3c3e8199d51532b0c067dc1595')
```

**Review by alalazo — CHANGES_REQUESTED** [review]
I think there's a typo, other than that LGTM

**Review by tldahlgren — APPROVED** [review]
Confirmed sha256 for both versions.

**Comment by vicentebolea** [comment]
@alalazo I think the PR got stuck somehow, I applied the requested changes but it is still waiting on that.

**Review by alalazo — APPROVED** [review]

**Pull request merged**